modyn/evaluator/internal/dataset/evaluation_dataset.py

```diff
@@ -81,7 +81,7 @@ def _setup_composed_transform(self) -> None:
             self._transform = transforms.Compose(self._transform_list)
 
     @retry(
-        stop=stop_after_attempt(5),
+        stop=stop_after_attempt(25),
```

Comment:
i would not really increase this to 25

Reply:
10 maybe? for evaluations, it's not harmful if they take longer because of more retries. Within the business logic it could make experiments produce weird results silently.

Reply:
I mean okay we can merge 10 but 25 seems to be a bit high to have in the main branch :D